vsphere/datadog_checks/vsphere/types/vim.py

```diff
@@ -0,0 +1,83 @@
+from datetime import datetime
```

Comment:
I know we generally advise against it, but would this file actually be a good candidate for a [stub file](https://mypy.readthedocs.io/en/stable/stubs.html)? Seems to be stubbing out an external library.

Reply:
I'm not sure to follow. Here we are just using `datetime` type.

Reply:
Oh yeah, I tried, it's quite cumbersome and more involved. So, not sure it worth doing so.

Reply:
What's the cumbersome aspect? Is it that you'd have to match the `pyvim` name and package structure?

Reply:
> Is it that you'd have to match the pyvim name and package structure?

Yes, right.

Reply:
Will have another try. Keep you updated.